ext/afform/core/managed/AfformConfirmationType.mgd.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use CRM_Afform_ExtensionUtil as E;
+
+// Adds option group for Afform.confirmation type
+return [
+  [
+    'name' => 'AfformConfirmationType',
+    'entity' => 'OptionGroup',
+    'update' => 'always',
+    'cleanup' => 'always',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'afform_confirmation_type',
+        'title' => E::ts('Afform Confirmation Type'),
+        'is_reserved' => TRUE,
+        'option_value_fields' => [
+          'name',
+          'label',
+          'icon',
+          'description',
+        ],
+      ],
+      'match' => ['name'],
+    ],
+  ],
+  [
+    'name' => 'AfformConfirmationType:redirect_to_url',
+    'entity' => 'OptionValue',
+    'cleanup' => 'always',
+    'update' => 'always',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'option_group_id.name' => 'afform_confirmation_type',
+        'name' => 'redirect_to_url',
+        'value' => 'redirect_to_url',
+        'label' => E::ts('Redirect to a URL'),
+        'icon' => 'fa-turn-up',
```

Comment:
Probably ought to add `is_reserved' => TRUE` here and below.

Reply:
done